src/main/java/uk/gov/hmcts/reform/bulkscan/constants/BulkScanConstants.java

```diff
@@ -212,8 +216,27 @@ public final class BulkScanConstants {
     public static final String ADDRESS_NOT_LIVED_FOR_FIVE_YEARS_MESSAGE =
             "(%s) has not lived at the current address for more than 5 years. Previous address(es)"
                     + " field (%s) should not be empty or null.";
+    public static final String PARENT_FIELD_VIOLATION_SHOULD_BE_EMPTY =
+            "Parent field violation - dependent field (%s) should be empty as it requires parent "
+                    + "field (%s) having value (%s).";
+
     // END C100 form section 2 validation fields
 
+    // C100 form section 8 & 9 validation fields
+    public static final String ASSESSMENT_BY_ADULT_LEARNING_TEAM_FIELD =
+            "assessmentByAdultLearningTeam";
+    public static final String FACTORS_AFFECTING_LITIGATION_CAPACITY_FIELD =
+            "factorAffectingLitigationCapacity";
+    public static final String FACTORS_AFFECTING_PERSON_IN_COURT_FIELD =
+            "factorsAffectingPersonInCourt";
+    public static final String INTERNATIONALELEMENT_JURISDICTIONISSUE =
+            "internationalElement_jurisdictionIssue";
+    public static final String INTERNATIONALELEMENT_REQUEST_CENTRAL_CONSULAR_AUTH =
+            "internationalElement_request_toCentral_or_Consular_authority";
+    public static final String INTERNATIONALELEMENT_RESIDENT_ANOTHER_STATE =
+            "internationalElement_Resident_of_another_state";
```

Comment:
can you move to bulkscanPRLconstants file  pls?

Reply:
Done.